llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -402,6 +403,42 @@ Expected<GroupRef> GroupRef::create(MCCASBuilder &MB,
   return get(B->build());
 }
 
+static Expected<uint64_t>
+materializeDebugInfoSection(MCCASReader &Reader,
+                            const SmallVector<cas::ObjectRef> &Refs,
+                            DebugInfoSectionRef &DebugInfoRef) {
+  // Find DebugAbbrevSectionRef and get it's section contents
+  SmallVector<char> AbbrevContents;
+  for (auto ID : Refs) {
+    auto Node = Reader.getObjectProxy(ID);
+    if (!Node)
+      return Node.takeError();
+    if (auto F = DebugAbbrevSectionRef::Cast(*Node)) {
```

Comment:
let's use an early continue here: https://llvm.org/docs/CodingStandards.html#use-early-exits-and-continue-to-simplify-code

though to be honest I think finding the `DebugAbbrevSectionRef` could be a function of its own and the loop above doesn't need to exist here